python/lsst/ts/observatory/control/auxtel/atcalsys.py

```diff
@@ -69,6 +70,14 @@ def __iter__(self) -> typing.Iterator[int]:
         )
 
 
+@dataclass
+class ATCalsysExposure:
```

Comment:
Please, add a docstring to this class explaining what it is for and also document the attributes, e.g.;

```
@dataclass
class ATCalsysExposure:
    """Store Exposure information for ATCalsys.

    Attributes
    ----------
    wavelength : float
        Wavelength of the calibration exposure, in nm.
    camera : float
        Camera exposure time, in sec.
    fiberspectrograph : float | None
        Fiber spectrograph exposure time, in sec.
        If None, skip fiber spectrograph acquisition.
    electrometer : float | None
        Electrometer exposure time, in sec.
        If None, skip electrometer acquisition.
    """
    ...
 ```